### PR TITLE
Logging

### PR DESCRIPTION
**Pull request by SathyaRupa**
- Make logs visible in terminal and stored in a file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -381,3 +381,4 @@ airport-app-backend
 golang-app-starter
 
 .env
+requests.log
```
Comment (vraravam) [inline .gitignore]: change this to `*.log`

File: middleware/logging.go [hunk 2]
```diff
@@ -10,47 +11,49 @@ import (
...
 		latency := time.Since(startTime)
 
-		subLogger := consoleLogger.With().Timestamp().
+		logMsg := "Request"
+		if len(ctx.Errors) > 0 {
+			logMsg = ctx.Errors.String()
+		}
+
+		status := ctx.Writer.Status()
+		requestPath := ctx.Request.URL.Path + "?" + ctx.Request.URL.RawQuery
```
Comment (vraravam) [inline middleware/logging.go]: please recheck the logic of how `requestPath` is calculated

**Review by vraravam — CHANGES_REQUESTED** [review]

File: server/routes.go [hunk 3]
```diff
@@ -42,8 +44,4 @@ func (srv *AppServer) setupRoutesAndMiddleware() {
 	srv.router.Use(middleware.AddSecurityHeaders(config.EnableTls))
 	srv.router.Use(middleware.HandleFaviconRequests())
 
-	if config.EnableDetailedRequestLogging {
```
Comment (vraravam) [inline server/routes.go:33]: Are you effectively removing this check? Is that intended?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: server/routes.go (hunk 3 above)
Comment (SathyaRupa) [reply server/routes.go:33]: yes. I intended to make the logs detailed for both valid and invalid endpoint, so removed the check

**Review by vraravam — APPROVED** [review]

**Pull request merged**